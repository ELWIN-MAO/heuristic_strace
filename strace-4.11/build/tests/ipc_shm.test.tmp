
mym_waitpid=102066

mym_waitpid=102066

mym_waitpid=102066

syscall_enter_mym_scno=59  sys_name=execve

mym_waitpid=102066

mym_waitpid=102066

syscall_exit_mym_scno=59  sys_name=execve

mym_waitpid=102066

syscall_enter_mym_scno=12  sys_name=brk

mym_waitpid=102066

syscall_exit_mym_scno=12  sys_name=brk

mym_waitpid=102066

syscall_enter_mym_scno=21  sys_name=access

mym_waitpid=102066

syscall_exit_mym_scno=21  sys_name=access

mym_waitpid=102066

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=102066

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=102066

syscall_enter_mym_scno=21  sys_name=access

mym_waitpid=102066

syscall_exit_mym_scno=21  sys_name=access

mym_waitpid=102066

syscall_enter_mym_scno=2  sys_name=open

mym_waitpid=102066

syscall_exit_mym_scno=2  sys_name=open

mym_waitpid=102066

syscall_enter_mym_scno=5  sys_name=fstat

mym_waitpid=102066

syscall_exit_mym_scno=5  sys_name=fstat

mym_waitpid=102066

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=102066

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=102066

syscall_enter_mym_scno=3  sys_name=close

mym_waitpid=102066

syscall_exit_mym_scno=3  sys_name=close

mym_waitpid=102066

syscall_enter_mym_scno=21  sys_name=access

mym_waitpid=102066

syscall_exit_mym_scno=21  sys_name=access

mym_waitpid=102066

syscall_enter_mym_scno=2  sys_name=open

mym_waitpid=102066

syscall_exit_mym_scno=2  sys_name=open

mym_waitpid=102066

syscall_enter_mym_scno=0  sys_name=read

mym_waitpid=102066

syscall_exit_mym_scno=0  sys_name=read

mym_waitpid=102066

syscall_enter_mym_scno=5  sys_name=fstat

mym_waitpid=102066

syscall_exit_mym_scno=5  sys_name=fstat

mym_waitpid=102066

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=102066

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=102066

syscall_enter_mym_scno=10  sys_name=mprotect

mym_waitpid=102066

syscall_exit_mym_scno=10  sys_name=mprotect

mym_waitpid=102066

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=102066

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=102066

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=102066

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=102066

syscall_enter_mym_scno=3  sys_name=close

mym_waitpid=102066

syscall_exit_mym_scno=3  sys_name=close

mym_waitpid=102066

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=102066

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=102066

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=102066

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=102066

syscall_enter_mym_scno=9  sys_name=mmap

mym_waitpid=102066

syscall_exit_mym_scno=9  sys_name=mmap

mym_waitpid=102066

syscall_enter_mym_scno=158  sys_name=arch_prctl

mym_waitpid=102066

syscall_exit_mym_scno=158  sys_name=arch_prctl

mym_waitpid=102066

syscall_enter_mym_scno=10  sys_name=mprotect

mym_waitpid=102066

syscall_exit_mym_scno=10  sys_name=mprotect

mym_waitpid=102066

syscall_enter_mym_scno=10  sys_name=mprotect

mym_waitpid=102066

syscall_exit_mym_scno=10  sys_name=mprotect

mym_waitpid=102066

syscall_enter_mym_scno=10  sys_name=mprotect

mym_waitpid=102066

syscall_exit_mym_scno=10  sys_name=mprotect

mym_waitpid=102066

syscall_enter_mym_scno=11  sys_name=munmap

mym_waitpid=102066

syscall_exit_mym_scno=11  sys_name=munmap

mym_waitpid=102066

syscall_enter_mym_scno=29  sys_name=shmget
shmget(IPC_PRIVATE, 1, 0600
mym_waitpid=102066

syscall_exit_mym_scno=29  sys_name=shmget

mym_scall_rval=0x2d000b
) = 2949131

mym_waitpid=102066

syscall_enter_mym_scno=5  sys_name=fstat

mym_waitpid=102066

syscall_exit_mym_scno=5  sys_name=fstat

mym_waitpid=102066

syscall_enter_mym_scno=12  sys_name=brk

mym_waitpid=102066

syscall_exit_mym_scno=12  sys_name=brk

mym_waitpid=102066

syscall_enter_mym_scno=12  sys_name=brk

mym_waitpid=102066

syscall_exit_mym_scno=12  sys_name=brk

mym_waitpid=102066

syscall_enter_mym_scno=31  sys_name=shmctl
shmctl(2949131, IPC_STAT, 
mym_waitpid=102066

syscall_exit_mym_scno=31  sys_name=shmctl
{shm_perm={uid=0, gid=0, mode=0600, key=0, cuid=0, cgid=0}, shm_segsz=1, shm_cpid=102066, shm_lpid=0, shm_nattch=0, shm_atime=0, shm_dtime=0, shm_ctime=1489666176}
mym_scall_rval=0
) = 0

mym_waitpid=102066

syscall_enter_mym_scno=31  sys_name=shmctl
shmctl(0, SHM_INFO, 
mym_waitpid=102066

syscall_exit_mym_scno=31  sys_name=shmctl
0x7ffcfd345a90
mym_scall_rval=0xb
) = 11

mym_waitpid=102066

syscall_enter_mym_scno=31  sys_name=shmctl
shmctl(2949131, SHM_STAT, 
mym_waitpid=102066

syscall_exit_mym_scno=31  sys_name=shmctl
0x7ffcfd345a90
mym_scall_rval=0x2d000b
) = 2949131

mym_waitpid=102066

syscall_enter_mym_scno=31  sys_name=shmctl
shmctl(2949131, IPC_RMID, 
mym_waitpid=102066

syscall_exit_mym_scno=31  sys_name=shmctl
NULL
mym_scall_rval=0
) = 0

mym_waitpid=102066

syscall_enter_mym_scno=1  sys_name=write

mym_waitpid=102066

syscall_exit_mym_scno=1  sys_name=write

mym_waitpid=102066

syscall_enter_mym_scno=231  sys_name=exit_group

mym_waitpid=102066
+++ exited with 0 +++
