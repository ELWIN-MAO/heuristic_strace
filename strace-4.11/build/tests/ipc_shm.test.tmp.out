shmget\(IPC_PRIVATE, 1, 0600\) += 2949131
shmctl\(2949131, (IPC_64\|)?IPC_STAT, \{shm_perm=\{uid=0, gid=0, mode=0600, key=0, cuid=0, cgid=0\}, shm_segsz=1, shm_cpid=102066, shm_lpid=0, shm_nattch=0, shm_atime=0, shm_dtime=0, shm_ctime=1489666176\}\) += 0
shmctl\(0, (IPC_64\|)?SHM_INFO, 0x7ffcfd345a90\) += 11
shmctl\(2949131, (IPC_64\|)?SHM_STAT, 0x7ffcfd345a90\) += 2949131
shmctl\(2949131, (IPC_64\|)?IPC_RMID, NULL\) += 0
lo IP Address 127.0.0.1
ens33 IP Address 192.168.159.137
IP Address 127.0.0.1
IP Address 192.168.159.137
